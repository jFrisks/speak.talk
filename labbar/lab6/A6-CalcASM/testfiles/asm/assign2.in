int main(){
  int x;
  int y = 10;
  x = y + 2;
  return f(y + x);
  print(42);
}

int f(int x){
  print(x);
  int y = 2;
  print(y);
}