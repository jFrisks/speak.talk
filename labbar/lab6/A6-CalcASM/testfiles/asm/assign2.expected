22
2
42